azkaban_cli/azkaban.py

```diff
@@ -366,3 +366,24 @@ def create(self, project, description):
         self.__catch_response_error(response, CreateError)
 
         logging.info('Project %s created successfully' % (project))
+
+    def delete(self, project):
+        """Delete command, intended to make the request to Azkaban and treat the response properly.
+
+        This method receives the project name, make the execute request to delete the project and
+        evaluate the response.
+
+        :param project: Project name on Azkaban
+        :type project: str
+        """
+
+        self.__check_if_logged()
+
+        _ = api.delete_request(
```

Comment:
```suggestion
        api.delete_request(
```

Reply:
I prefer like this, but just a suggestion :P (and if you had no specific reason to do it your way, of course)

Reply:
Fixed in new commit https://github.com/globocom/azkaban-cli/tree/add_delete_request